### PR TITLE
Add integrate `spark-rewards`

### PR DESCRIPTION
**Pull request by juliangruber**
https://github.com/space-meridian/roadmap/issues/161

### CODE REVIEW

**Review by bajtos — APPROVED** [review]
LGTM.

I don't see any tests for this change, what's your plan to verify the code works as intended?

File: observer/lib/observer.js [hunk 1]
```diff
@@ -35,8 +35,24 @@ export const observeTransferEvents = async (pgPoolStats, ieContract, provider) =
...
 }
 
+const getScheduledRewards = async (address, ieContract) => {
+  const [fromContract, fromSparkRewards] = await Promise.all([
+    ieContract.rewardsScheduledFor(address),
+    (async () => {
+      const res = await fetch(
+        `https://spark-rewards.fly.dev/scheduled-rewards/${address}`
+      )
+      const json = await res.json()
```
Comment (bajtos) [inline observer/lib/observer.js:45]: How do you handle 4xx and 5xx responses? What if the server responds with 5xx and the response body provides error details? Should you check `res.ok`?

In other places, I had a great experience with the helper `assertOkResponse` that reads the error response body and includes it in `err.message`.

Comment (juliangruber) [reply observer/lib/observer.js:45]: see reply to same question in other PR

Comment (bajtos) [reply observer/lib/observer.js:46]: Link to that reply: https://github.com/filecoin-station/core/pull/606#discussion_r1791853853

**Pull request merged**